dataland-frontend/src/router/index.ts

```diff
@@ -22,11 +25,21 @@ const routes = [
   {
     path: "/",
     name: "Welcome to Dataland",
+    component: NewLandingPage,
+  },
+  {
+    path: "/lp",
+    name: "New Landing Page",
     component: WelcomeDataland,
     props: {
       isMobile: /Android|webOS|iPhone|iPad|iPod|BlackBerry|Windows Phone|IEMobile|OperaMini/i.test(navigator.userAgent),
     },
```

Comment:
These two routes should probably be one route now, and be connected to the new landing page.

The old landing page should be removed.

Reply:
solved